mk/config.mk

```diff
@@ -543,6 +543,10 @@ CFG_DRIVERS_CLK_EARLY_PROBE ?= n
 $(call force,CFG_DRIVERS_DT_RECURSIVE_PROBE,n,Mandated by CFG_DT_DRIVER_EMBEDDED_TEST)
 endif
 
+# CFG_WITH_STATS when enabled embeds PTA stats service to allow non-secure
```

Comment:
Perhaps spell out statistics instead of stats here

Reply:
Ok, i'll do.